### DISCUSSION TITLE
Podcini needs 20 testers for publishing on Google Play

### DISCUSSION THREAD

**Discussion by XilinJia** [XilinJia/Podcini#26] [Announcements]
Please help testing it by joining the Google group: https://groups.google.com/g/podcini and download the app from Google play.  The version is 4.4.3

**Comment by summersab** [comment]
I used every account I have to get you to 19 and got a friend to bring you up to 20. What's next?

After that, I had an idea for a way to potentially grow your user and developer base. First, post a release announcement on places like Reddit or maybe XDA. Lots of developers do that, and it gets attention.

Also, you could contact sites that have done reviews of AmtennaPod or made lists of the "best podcast apps." There are several FOSS sites as well as commercial blogs that often do this - just do a few searches to find them.

**Comment by XilinJia** [comment]
Thank you.
I will re-initiate the publishing-on-Google process once Podcini 6 gets stable.
